doi-requests-commons/src/test/java/no/unit/nva/doi/requests/service/impl/DynamoDBDoiRequestsServiceTest.java

```diff
@@ -65,6 +64,7 @@
     public static final String INVALID_USERNAME = "invalidUsername";
     public static final DoiRequestStatus INITIAL_DOI_REQUEST_STATUS = REQUESTED;
     public static final DoiRequestStatus NEW_DOI_REQUEST_STATUS = APPROVED;
+    public static final AWSCredentialsProvider EMPTY_CREDENTIALS = null;
```

Comment:
Move this to the Factory class, make it protected, and use it here.

Reply:
fixed